docs/semgrep-code/triage-remediation.md

```diff
@@ -132,10 +135,20 @@ To **reopen multiple findings** in the **No grouping** view, follow these steps:
 
 </details>
 
-## Ignore findings through GitHub PR comments
+## Ignore findings through PR and MR comments
+
+Triage your Semgrep AppSec Platform findings displayed as comments in GitHub PRs and GitLab MRs by replying with another comment.
 
-Triage your Semgrep AppSec Platform findings displayed as comments in GitHub PRs by replying with another comment.
+<Tabs
+    defaultValue="current"
+    values={[
+        {label: 'GitHub', value: 'current'},
+        {label: 'GitHub (beta)', value: 'gh'},
+        {label: 'GitLab (beta)', value: 'gl'}
+    ]}
+>
 
+<TabItem value='current'>
 :::info Prerequisites
 - A **private** GitHub.com repository. This feature is not enabled for public GitHub.com repositories or GitHub Enterprise public and private repositories.
```

Comment:
```suggestion
- A **private** GitHub Free or Team cloud-hosted repository. This feature is not enabled for public GitHub repositories or GitHub Enterprise public and private repositories.
```
Just trying to use the specific plan terms as provided in https://github.com/pricing, which is our source of truth.